### PR TITLE
Contract verify

### PR DESCRIPTION
**Pull request by krofax**

### CODE REVIEW

File: docs/dev/guide/contract-verification.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## What is contract verification?
 
-The bytecode generated by the Solidity compiler is essentially deployed when a Smart Contract is deployed on a network. The original Solidity code is not retained in any way.
+When you deploy a Smart Contract on a network, you’re essentially deploying the bytecode produced by the Solidity compiler. The original Solidity code is not retained in any way.
```
Comment (barakshani) [inline docs/dev/guide/contract-verification.md:5]: why capitalize Smart Contract?

File: docs/dev/guide/contract-verification.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 
 Verifying a contract entails determining whether the on-chain bytecode generated from the Smart Contract source code complies. If it does, we can state that we have confirmed the bytecode's source code's integrity. Hence the name – verification.
```
Comment (barakshani) [inline docs/dev/guide/contract-verification.md:7]: I don't understand this part? What do you mean compiles?
Verification makes sure that some external code (e.g. from GitHub) is the same code as the deployed code

File: docs/dev/guide/contract-verification.md [hunk 3]
```diff
@@ -72,10 +72,35 @@ To verify the contract, enter the following details;
...
 - Constructor arguments: If Constructor Arguments are required by the contract, you should add them here, but in [ABI hex encoded form](https://solidity.readthedocs.io/en/develop/abi-spec.html).
-- Click the **Verify Smart Contract button**.
+  
+<br>
+
+**Note:** To obtain the Constructor arguments, after deploying your contract, include this 
+<br>
+
+```
+console.log("constructor args:" + greeterContract.interface.encodeDeploy([greeting]));
```
Comment (barakshani) [inline docs/dev/guide/contract-verification.md:85]: This is specifically for the greeterContract that we have in our Hello World tutorial. It won't always work

Comment (barakshani) [reply docs/dev/guide/contract-verification.md:85]: we also need to explain that then the args will be output on the screen on contract deployment

Comment (krofax) [reply docs/dev/guide/contract-verification.md:85]: Noted.

File: docs/dev/guide/contract-verification.md (hunk 1 above)
Comment (krofax) [reply docs/dev/guide/contract-verification.md:5]: Noted, I'll correct this.

File: docs/dev/guide/contract-verification.md (hunk 2 above)
Comment (krofax) [reply docs/dev/guide/contract-verification.md:7]: Rephased to this 

Contract verification is ensuring the correctness, security, and safety of the smart contract implementations which allows smart contract developers to prove and publish the source code of the contracts deployed on-chain.

**Pull request merged**